### PR TITLE
Rename ingest processor supports overriding target field if exists

### PR DESCRIPTION
**Pull request by gaobinlong**
### Description
In 2.14.0, we'll add a new parameter override_target for the existing rename ingest processor, we need to add the parameter in the documentation of the rename ingest processor.

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/6949

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by vagimeli — APPROVED** [review]
Doc review complete

File: _ingest-pipelines/processors/rename.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The following table lists the required and optional parameters for the `rename`
 `field`  | Required  | The field name containing the data to be removed. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). |
 `target_field`  | Required  | The new name of the field. Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets). |
 `ignore_missing`  | Optional  | Specifies whether the processor should ignore documents that do not contain the specified `field`. If set to `true`, the processor does not modify the document if the `field` does not exist. Default is `false`. |
+`override_target`  | Optional  | Determines the behavior when `target_field` already exists in the document. If set to `true`, the processor overwrites the existing `target_field` value with the new value. Default is `false`. |
```
Comment (natebower) [inline _ingest-pipelines/processors/rename.md:37]: Can we clarify the type of behavior ("X behavior") or use something like "method used"?

**Review by natebower — COMMENTED** [review]
@gaobinlong @vagimeli Just one comment. Thanks!

File: _ingest-pipelines/processors/rename.md (hunk 1 above)
Comment (vagimeli) [reply _ingest-pipelines/processors/rename.md]: Revised to read: `Determines what happens when `target_field` exists in the document.`

**Pull request merged**